### PR TITLE
Polish upkeep bullets

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

File: R/upkeep.R [hunk 1]
```diff
@@ -118,86 +122,56 @@ use_tidy_upkeep_issue <- function(year = NULL) {
...
-      todo("
-        `usethis::use_testthat(3)` and upgrade to 3e, \\
-        [testthat 3e vignette](https://testthat.r-lib.org/articles/third-edition.html)"),
-      todo("
-        Align the names of `R/` files and `test/` files for workflow happiness.
-        The docs for `usethis::use_r()` include a helpful script.
-        `usethis::rename_files()` may be be useful."),
+      todo("`usethis::use_package_doc()`"),
+      todo("`usethis::use_testthat(3)`"),
+      todo("Align the names of `R/` files and `test/` files"),
```
Comment (jennybc) [inline R/upkeep.R:153]: In general, I like having detail, links, etc. But I can also imagine this is about making the items for the distant past more lean, yeah? And saving detail for the items we are more likely to be engaging with in the year of our lord 2023?

File: tests/testthat/_snaps/upkeep.md [hunk 2]
```diff
@@ -3,71 +3,57 @@
...
-      * [ ] Prefer `pak::pak("org/pkg")` over `devtools::install_github("org/pkg")` in README
+      * [ ] Use `pak::pak("org/pkg")` in README
       * [ ] Consider running `use_tidy_dependencies()` and/or replace compat files with `use_standalone()`
-      * [ ] `use_standalone("r-lib/rlang", "types-check")` instead of home grown argument checkers
+      * [ ] Use cli errors or [file an issue](new) if you don't have time to do it now
+      * [ ] `use_standalone("r-lib/rlang", "types-check")` instead of home grown argument checkers;
+      or [file an issue](new) if you don't have time to do it now
       * [ ] Add alt-text to pictures, plots, etc; see https://posit.co/blog/knitr-fig-alt/ for examples
       
+      ### Eternal
```
Comment (jennybc) [inline tests/testthat/_snaps/upkeep.md:48]: Good idea 💡

**Review by jennybc — APPROVED** [review]

File: R/upkeep.R (hunk 1 above)
Comment (hadley) [reply R/upkeep.R:154]: Yeah, exactly.

File: tests/testthat/_snaps/upkeep.md [hunk 3]
```diff
@@ -36,7 +36,7 @@
       * [ ] Update email addresses *@rstudio.com -> *@posit.co
       * [ ] Update copyright holder in DESCRIPTION: `person("Posit Software, PBC", role = c("cph", "fnd"))`
       * [ ] `Run devtools::document()` to re-generate package-level help topic with DESCRIPTION changes
```
Comment (jennybc) [inline tests/testthat/_snaps/upkeep.md:38]:
```suggestion
      * [ ] Run `devtools::document()` to re-generate package-level help topic with DESCRIPTION changes
```

Not in the diff but something I just noticed.

**Review by jennybc — APPROVED** [review]

**Pull request merged**